CONTRIBUTING.md

```diff
@@ -26,12 +26,45 @@ Please try to spend some time on a good title and description of the issue.
 
 ## Pull requests
 
-If it is not a fork branch, please use the following format for your branch: `{gatsby,drupal}/{feature,bugfix,content}/slug`. 
+If it is not a fork branch, please use the following format for your branch: `{gatsby,drupal}/{feature,bugfix,content}/slug`.
 
 All pull requests should undergo code review. The code review can be performed by any volunteer who has been active in the project for a long time.
 
 A Pull Request has to pass validation by Github Actions (Triggered Automatically)
 
+## Branching Workflow
+
+### Branch overview
+
+* `master`, main integration and testing branch
```

Comment:
this is super confusing, I'd suggest sticking with develop for integration and testing and master/main for release branch

Reply:
Well, I have previously suggested that, but @kanei points out that this would change the main branch for contributors, which would also be confusing 🤷‍♂️

Reply:
(The discussion [continues in Slack](https://cesko-digital.slack.com/archives/C01DVGTNRUL/p1605191387301000?thread_ts=1604786859.494100&cid=C01DVGTNRUL), we will resolve the issue there.)

Reply:
Yeah exactly, I agree with the fact that e.g. once you start creating PRs, the default branch to merge to is `master`, the same when you checkout the repo.. its `master` again. and we will have always the master deployed to https://covidportal.dev/, so that would be our default flow.

And once in a while when we plan a `release`, we would merge `master` to `maint`.

Reply:
But in the end, i dont really care as long as prod and staging will be separated :D

Reply:
after you change the default branch in repo, none of what you wrote is true... The only change is that you need to change workflow in vercel, which should be one click

Reply:
Resolved in Slack, we have voted to have `develop` as the main working branch and `master` for releases.